integration/client/client_unix_test.go

```diff
@@ -30,7 +30,7 @@ var (
 	testImage             = images.Get(images.BusyBox)
 	testMultiLayeredImage = images.Get(images.VolumeCopyUp)
 	shortCommand          = withProcessArgs("true")
-	longCommand           = withProcessArgs("/bin/sh", "-c", "while true; do sleep 1; done")
+	longCommand           = withProcessArgs("/bin/sh", "-c", "while true; do sleep 1d; done")
```

Comment:
What about this instead?

```suggestion
	longCommand           = withProcessArgs("/bin/sh", "-c", "sleep inf")
```

Reply:
```
 sudop ctr run --rm -t docker.io/library/busybox:1.28 testing sleep
BusyBox v1.28.4 (2018-05-22 17:00:17 UTC) multi-call binary.

Usage: sleep [N]...

Pause for a time equal to the total of the args given, where each arg can
have an optional suffix of (s)econds, (m)inutes, (h)ours, or (d)ays
```

The `inf` is only available to new version of `sleep`. I keep it in `1d` because it is long enough.
Will update it when the image is available.

Reply:
I'm not sure at which version it was introduced, but docker.io/library/busybox:1.36 supports `sleep inf`.  Should we move to that?

Reply:
Sounds good. Will do after https://github.com/containerd/containerd/actions/runs/4036082318/jobs/6938428900

Reply:
Updated! PTAL. Thanks!